src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/set_interior_vehicle_data_request.cc

```diff
@@ -320,13 +501,33 @@ bool SetInteriorVehicleDataRequest::AreAllParamsReadOnly(
   return true;
 }
 
+bool CheckReadOnlyParamsForAudio(
+    const smart_objects::SmartObject& module_type_params) {
+  if (module_type_params.keyExists(message_params::kEqualizerSettings)) {
+    const auto& equalizer_settings =
+        module_type_params[message_params::kEqualizerSettings];
+    auto it = equalizer_settings.asArray()->begin();
+    for (; it != equalizer_settings.asArray()->end(); ++it) {
+      if (it->keyExists(message_params::kChannelName)) {
+        LOG4CXX_DEBUG(logger_, " READ ONLY parameter: ");
```

Comment:
Something should go after the `:`, it implies there is more information to the statement

Reply:
@jacobkeeler fixed c8b4d07a019e5f8d89f81bd0d6604270f43452a9